index.d.ts

```diff
@@ -189,6 +189,31 @@ export declare interface SamplingRule {
   name?: string | RegExp
 }
 
+/**
+ * Span sampling rules to ingest single spans where the enclosing trace is dropped
+ */
+export declare interface SpanSamplingRule {
+  /**
+   * Sampling rate for this rule. Will default to 1.0 (always) if not provided.
+   */
+  sampleRate?: Number
+
+  /**
+   * Maximum number of spans matching a span sampling rule to be allowed per second.
+   */
+  maxPerSecond: Number
```

Comment:
Did you forget a `?` here ? (just asking because I always make that mistake, and it looks like it defaults to Infinity)